metricbeat/module/vsphere/host/host_test.go

```diff
@@ -37,14 +37,16 @@ func TestFetchEventContents(t *testing.T) {
 	ts := model.Service.NewServer()
 	defer ts.Close()
 
-	f := mbtest.NewEventsFetcher(t, getConfig(ts))
-
-	events, err := f.Fetch()
+	f := mbtest.NewReportingMetricSetV2Error(t, getConfig(ts))
+	events, errs := mbtest.ReportingFetchV2Error(f)
 	if err != nil {
```

Comment:
I think this line should be removed as it checks an error from above.

Reply:
Good catch!